tsup.config.ts

```diff
@@ -5,15 +5,16 @@ import { defineConfig } from 'tsup';
 
 export default defineConfig((options: Options) => ({
```

Comment:
i think this type and the import above are redundant:

```suggestion
export default defineConfig((options) => ({
```